src/streaming/constants/Constants.js

```diff
@@ -246,6 +246,22 @@ export default {
         EVENT: 'event'
     },
 
+    /**
+     *  @constant {string} CMCD_EVENTS specifies all the available events for CMCD.
```

Comment:
It is necessary to specify that the CMCD events are used by event mode. Maybe something like this:

```suggestion
     *  @constant {string} CMCD_EVENTS specifies all the available events for CMCD event mode.
```

Reply:
Added event mode clarification